bin/lookup_evolve.py

```diff
@@ -115,16 +115,16 @@ def crossover(self, other):
         # Assuming that the number of states is the same
         new_table = self.crossover_tables(self.table, other.table)
         return LookerUpParams(self.plays, self.op_plays, self.op_start_plays,
-                              list(self.initial_actions), self.mutation_rate,
+                              list(self.initial_actions), self.mutation_probability,
                               new_table)
 
     def __repr__(self):
         return "{}:{}:{}:{}:{}".format(
             self.plays,
             self.op_plays,
             self.op_start_plays,
-            ''.join(self.initial_actions),
-            ''.join([v for k, v in sorted(self.table.items())])
+            actions_to_str(self.initial_actions),
+            actions_to_str([v for k, v in self.table.items()])
```

Comment:
I think we need to sort these guys (via actions like you suggested above). Otherwise these representations won't be unique, and reversing `__repr__` won't work in a stable manner.

Reply:
Once https://github.com/Axelrod-Python/Axelrod/pull/1199 is in and a new release of Axelrod done (I'll do it as soon as) we could bump https://github.com/Axelrod-Python/axelrod-dojo/blob/master/requirements.txt to keep track of the fact that we need the ordered actions?

Reply:
SGTM